lib/reek/smell_detectors/missing_safe_method.rb

```diff
@@ -48,20 +48,20 @@ def self.contexts # :nodoc:
       #
       def sniff
         context.node_instance_methods.select do |method_sexp|
-          prima_donna_method?(method_sexp)
+          missing_safe_method?(method_sexp)
         end.map do |method_sexp|
           name = method_sexp.name
           smell_warning(
             context: context,
             lines: [method_sexp.line],
-            message: "has prima donna method '#{name}'",
+            message: "has missing safe method '#{name}'",
```

Comment:
The missing safe method is `name` without the bang. Perhaps `has dangerous method '#{name}' without a safe version`.

Reply:
I was thinking the wording there (and in a few other places) was awkward, but was attempting to limit the scope of changes I made. Can amend this and other awkward phrasings.

Reply:
I like @mvz suggestion :)